caf-audit-binding-elasticsearch/src/main/java/com/hpe/caf/auditing/elastic/ElasticAuditRestHighLevelClientFactory.java

```diff
@@ -19,7 +19,6 @@
 import org.apache.http.HttpHost;
```

Comment:
Should we not have renamed this class?
ElasticAuditRestHighLevelClientFactory > OpenSearchTransportFactory